### DISCUSSION TITLE
Jaeger ingester not promoting kafka offset

### DISCUSSION THREAD

**Discussion by nata7che** [jaegertracing/jaeger#3364] [Q&A]
Hi all, I’m experiencing something weird. 
I have an `ingester` running and writing to ES. Spans are written properly. 

However, the group’s offset is not promoted AT ALL, and lag starts to build up. I tried resetting the group’s offset in a bunch of ways but this behavior persists. Anyone come across this behavior and can assist?

tnx.

**Comment by albertteoh** [comment]
Hi @nata7che, I haven't seen this before, but what I'd suggest is to have a look at the metrics that the ingester emits, which may shed some more light on the problem:

- "messages"
- "current-offset"
- "offset-lag"
- "partition-close"
- "partition-start"

The metrics are available on the ingester's port `14270`: https://www.jaegertracing.io/docs/1.27/deployment/#ingester
